src/web-ui/xdn.config.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
```

Comment:
All new source files require license header. See an existing .js source file for header to use.

Reply:
Added